NuGet.Docs/Release-Notes/NuGet-3.3.md

```diff
@@ -0,0 +1,45 @@
+
+# NuGet 3.3 Release Notes
+
+[NuGet 3.2.1 Release Notes](nuget-3.2.1) 
+
+NuGet 3.3 was released November 30, 2015 with a significant number of user interface updates and command-line features as well as a collection of useful fixes to the NuGet clients.
+
+## New Features
+
+* Credential Providers have been introduced that allow all NuGet clients to be able to work seamlessly with an authenticated feed.  [Instructions on how to install the Visual Studio Online credential provider ](http://docs.nuget.org/Consume/Credential-Providers) and configure the NuGet clients to use it are available on NuGet Docs.
```

Comment:
credential providers where added for the command line only, and vso credential support was added for the vsix